### DISCUSSION TITLE
3 Installing options : what should administrator take in account ?

### DISCUSSION THREAD

**Discussion by Keepshango** [AzuraCast/AzuraCast#4444] [Q&A]
Hi,

Based on informations provided in the installation page, Azuracast can be installed in three different ways :

Installing ((option 1)
Unattended Installation : release or stable channel (option 2 and 3)

**/// Q1 ///**
But which one makes it easier to webradio administrator to be in control for the management of Azuracast security, updates and upgrades, customization, collaboration, and so on ?

**/// Q2 ///**
Is the choice between release channel and stable channel is a matter of being or not a developper ? 

Regards

**Comment by BusterNeece** [comment]
@Keepshango To answer your question, there are really just two installation methods that you can choose from:

 - Docker installation takes advantage of the Docker containerization tool to allow us to prebuild images that have all the necessary software installed and configured to work together perfectly. For you as a user, it's _much_ faster to update and since everything's in a "sandbox" it's less likely to conflict with other software on the computer. If you're using AzuraCast basically "out of the box" with little customization, this installation method will work extremely well for you; we're adding an increasing number of tools to the application itself to help make Docker knowledge less of a requirement for common functions.

- Ansible installation actually directly modifies all the software directly on your server to allow it to run AzuraCast. All the same software is installed as in the Docker installation, but each of those libraries could conflict with existing software on your system, and it's not recommended to run anything else alongside AzuraCast when you use this installation method. On the other hand, you have very direct access to the filesystem and can modify anything without much issue, which can be very useful for tech-savvy "power users" who are comfortable with server administration.

Your question about which release channel to use is actually far less important of a decision when installing. Regardless of whether you choose the Stable or Rolling release version, you'll have access to the exact same web interface, the same software and the same tools. The "Stable" channel is just updated less frequently and is only updated once we've had a long period without any reported issues on our "Rolling Release" channel, which has updates several times a day but lets you take advantage of newer fixes. You can even switch from one channel to the other (you can always switch from Stable to Rolling, but should only switch from Rolling to Stable when Stable is newer than your Rolling version, so you're upgrading _forward_ and not _backward_).

Basically, regardless of your installation choices you'll get the very same AzuraCast. The question is whether you prefer a more stable version with less frequent new feature releases, or if you're a "power user" who prefers (or, for technical reasons, has to) use the Ansible installer instead of Docker.
